exporter/signalfxexporter/translation/constants.go

```diff
@@ -220,5 +220,12 @@ translation_rules:
   mapping:
     free: df_inodes.free
     used: df_inodes.used
+    used: memory.used
+
+# memory utilization
+- action: divide_metrics
+  metric_name: memory.utilization
+  mapping:
```

Comment:
This rule reuses the `mapping` field to avoid creating a new field, but I'm on the fence about it. The way it uses the mapping field is by requiring only one key-value pair, where the key is the numerator metric, and the value is denominator.

Reply:
What do you think of more generic approach?
action name could be `calculate_new_metric`
it'll take `metric_name` as a name of new metric
and new fields:
`operand1_metric`
`operator` (would be only `divide` atm)
`operand2_metric`
?

Reply:
I was on the fence about it before, but I think it's a worthwhile change. Will update unless I hear otherwise.

Reply:
> could you please cover the invalid input to increase the coverage?

Yes, done.